src/main/java/io/vertx/openapi/contract/MediaType.java

```diff
@@ -33,11 +34,19 @@ public interface MediaType extends OpenAPIObject {
   String MULTIPART_FORM_DATA = "multipart/form-data";
   String APPLICATION_HAL_JSON = "application/hal+json";
   String APPLICATION_OCTET_STREAM = "application/octet-stream";
+  String TEXT_PLAN = "text/plain";
+  String TEXT_PLAIN_UTF8 = TEXT_PLAN + "; charset=utf-8";
   List<String> SUPPORTED_MEDIA_TYPES = Arrays.asList(APPLICATION_JSON, APPLICATION_JSON_UTF8, MULTIPART_FORM_DATA,
-    APPLICATION_HAL_JSON, APPLICATION_OCTET_STREAM);
+    APPLICATION_HAL_JSON, APPLICATION_OCTET_STREAM, TEXT_PLAN, TEXT_PLAIN_UTF8);
+
+  Pattern VENDOR_SPECIFIC_JSON = Pattern.compile("^[^/]+/vnd\\.[\\w.-]+\\+json$");
```

Comment:
move it to another (package private) class, or annotated with `@GenIgnore`

Reply:
Stupid question ... by putting @GenIgnore instead of @VertxGen I am actually doing the opposite of the initial intention, or? Doesn't it really matter?

Reply:
@vietj before I move forward, anything on the above question?

Reply:
not sure I understand your question : MediaType is annoated with VertxGen to trigger code generation and be available in RxJava or Mutiny, using GenIgnore ensures the code generator ignores this declaration of MediaType

Reply:
@vietj yeah my question was ... if we move from `VertxGen` to `GenIgnore` we are just changing the initial goal which was triggering code generation, is that ok anyway?

Reply:
> but I think here we need to keep it annotated and generated because it can be used by RxJava applications

@vietj So just to avoid confusion, you are actually suggesting to do "move it to another (package private) class".
Just to understand what exactly you mean by that, are you envisaging a dedicated static class annotated with `GetIgnore` which just provides the pattern and the functionality of matching what we need? Is there any pattern you follow in the Vert.x stack doing this where a pattern is needed I should follow to be more consistent?

Reply:
I just moved forward pulling the pattern out in a new interface with `@GenIgnore`, let me know if it works for you to fix the issue.